modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -175,22 +222,7 @@ const Actions = ({fdsView, namespace, spritemap}: IFDSViewSectionInterface) => {
 
 	return (
 		<ClayLayout.ContainerFluid>
-			<ClayBreadcrumb
-				className="my-2"
-				items={[
-					{
-						active: activeSection === SECTIONS.ACTIONS,
-						label: Liferay.Language.get('actions'),
-						onClick: () => setActiveSection(SECTIONS.ACTIONS),
-					},
-					{
-						active: activeSection === SECTIONS.NEW_ITEM_ACTION,
-						label: Liferay.Language.get('new-item-action'),
-						onClick: () =>
-							setActiveSection(SECTIONS.NEW_ITEM_ACTION),
-					},
-				]}
-			/>
+			<ClayBreadcrumb className="my-2" items={getBreadcrumbItems()} />
```

Comment:
Is it necessary that `getBreadcrumItems` is a function? Seems like it could just be a static set of items?

Reply:
I thought it'd be a cleaner way to handle the conditional items, unless you can see a clean way of doing that here? 

https://github.com/liferay-frontend/liferay-portal/blob/ce6a9105e231ed0bd70dd2df5ba6bc1841d513eb/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx#L71-L84

I suppose we could do something like

```
items={[
	{
		active: activeSection === SECTIONS.ACTIONS,
		label: Liferay.Language.get('actions'),
		onClick: () => setActiveSection(SECTIONS.ACTIONS),
	},
	{
		active: activeSection === SECTIONS.NEW_ITEM_ACTION || activeSection === SECTIONS.EDIT_ITEM_ACTION,
		label: activeSection === SECTIONS.NEW_ITEM_ACTION ? Liferay.Language.get('new-item-action') : initialActionFormValues?.label || '',
	}
]}
```

But it doesn't read as well. Plus it could get messier as we add the creation items section.

Reply:
My thought was to just omit the wrapping function since it will be called during the lifecycle, so I think we can just do this instead

```tsx
const breadcrumbItems: React.ComponentProps<typeof ClayBreadcrumb>['items'] = [
	{
		active: activeSection === SECTIONS.ACTIONS,
		label: Liferay.Language.get('actions'),
		onClick: () => setActiveSection(SECTIONS.ACTIONS),
	},
];

if (activeSection === SECTIONS.NEW_ITEM_ACTION) {
	breadcrumbItems.push({
		active: true,
		label: Liferay.Language.get('new-item-action'),
		onClick: () => setActiveSection(SECTIONS.NEW_ITEM_ACTION),
	});
}

if (activeSection === SECTIONS.EDIT_ITEM_ACTION) {
	breadcrumbItems.push({
		active: true,
		label: initialActionFormValues?.label || '',
	});
}

// ...

items={breadcrumbItems}

```